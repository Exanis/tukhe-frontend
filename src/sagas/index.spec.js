import rootSaga from './index';

it('Should have all the correct listeners', () => {
    const gen = rootSaga();

    expect(gen.next()).toMatchSnapshot();
    expect(gen.next().done).toBeTruthy();
});
