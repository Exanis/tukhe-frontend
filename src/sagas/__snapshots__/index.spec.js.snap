// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should have all the correct listeners 1`] = `
Object {
  "done": false,
  "value": Object {
    "@@redux-saga/IO": true,
    "combinator": true,
    "payload": Array [
      Object {},
      Object {},
      Object {},
    ],
    "type": "ALL",
  },
}
`;
